corehq/apps/callcenter/utils.py

```diff
@@ -61,19 +63,24 @@ def is_midnight_for_domain(midnight_form_domain, error_margin=15, current_time=N
 
 
 def get_call_center_domains():
-    result = (
-        DomainES()
-        .is_active()
-        .filter(filters.term('call_center_config.enabled', True))
-        .source([
-            'name',
-            'default_timezone',
-            'call_center_config.case_type',
-            'call_center_config.case_owner_id',
-            'call_center_config.use_user_location_as_owner',
-            'call_center_config.use_fixtures'])
-        .run()
-    )
+    try:
+        result = (
+            DomainES()
+            .is_active()
+            .filter(filters.term('call_center_config.enabled', True))
+            .source([
+                'name',
+                'default_timezone',
+                'call_center_config.case_type',
+                'call_center_config.case_owner_id',
+                'call_center_config.use_user_location_as_owner',
+                'call_center_config.use_fixtures'])
+            .run()
+        )
+    except ESError as e:
+        if 'IndexMissingException' in str(e) and settings.UNIT_TESTING:
```

Comment:
really not a fan of this, open to suggestions.

Reply:
I've always used https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/tests/base_test.py#L50-L52

Reply:
the challenge is that this get's called in the  migration. I could add optional mocking to the migration but not a great option either.